integration-tests/test_184.py

```diff
@@ -1,3 +1,13 @@
+#
+# -*- coding: utf-8 -*-
```

Comment:
The leading blank comment on top looks a bit weird, but I looked it up and `coding` can indeed be on the first **or** second line, so I guess this is fine: [PEP263](https://www.python.org/dev/peps/pep-0263/)

Reply:
We can get rid of the leading blank line. It's not necessary.

Personally, I avoid the shebang unless the script is executable, simply because it is unnecessary otherwise.

> I'd prefer to include the Python shebang always (any harm?).

Well, there's no harm (other than it being an eyesore). But consider this analogy: there's no harm in making **all** of our modules executable (chmod u+x), but we don't do it for the same reason (lack of necessity).